DataFormats/GEMDigi/src/GEMPadDigi.cc

```diff
@@ -19,7 +26,14 @@ bool GEMPadDigi::operator<(const GEMPadDigi& digi) const {
     return digi.bx() < bx_;
 }
 
-bool GEMPadDigi::isValid() const { return bx_ != -99 and pad_ != 65535; }
+bool GEMPadDigi::isValid() const {
+  // ME0 and GE11 have currently the same invalid pad
```

Comment:
since there are separate enums for them, maybe better to check explicitly (in case it changes in the future)